src/theme/components/modal.scss

```diff
@@ -23,25 +23,25 @@
   animation-fill-mode: forwards;
   animation-iteration-count: 1;
 
-  ion-backdrop {
+  .ion-page {
+    position: static;
+    contain: unset;
+    align-items: center;
+    padding: $size-xl 0;
+    min-height: 100vh;
+    justify-content: center;
+  }
+
+  &::part(backdrop) {
     display: none;
   }
 
-  .modal-wrapper {
```

Comment:
Modal's layout on mobile is broken: 
<img width="427" alt="image" src="https://user-images.githubusercontent.com/62751885/172653642-c6a77b4e-95e9-48a9-9bf9-bc50e1a8e583.png">